command.c

```diff
@@ -2741,11 +2743,56 @@ static void perform_external_cmd(int call, char *ext_cmd)
       memmove(cmd_args + 1, cmd_args, alen);
       cmd_args[0] = ' ';
       }
+    exefile = fopen(full_cmd,"rb");
```

Comment:
Hmm, but this and block below
are executed even from loadfix
itself!
But I guess I'll fix that myself.